src/assets/styles/_mixins.scss

```diff
@@ -0,0 +1,16 @@
+@mixin Small {
+    @media (min-width: 530px) {
```

Comment:
For later might be helpful to make the breakpoints as variables in the variables.scss. e.g.:

```
  $breakpoint-small: 530px;
  $breakpoint-medium: 882px;
  $breakpoint-large: 1024px;
```